src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -1015,7 +1023,23 @@ static int rbd_nbd(int argc, const char *argv[])
   int r;
   Config cfg;
   vector<const char*> args;
+
+  code_environment_t code_env = CODE_ENVIRONMENT_UTILITY;
+  int env_flag = 0;
+
   argv_to_vec(argc, argv, args);
+  env_to_vec(args);
+
+  for (auto &temp_args : args) {
+	if (strcmp(temp_args, "map") == 0) {
```

Comment:
We discussed issues with such manual parsing earlier. Consider a case when a user has a pool named "map" and specified `--rbd-default-pool map` in the command line.

Reply:
It thought about it, but I didn't think it is a bug:
For map command, the user must input "rbd-nbd map".  Even for the case "--rbd-default-pool map", it should be "rbd-nbd map --rbd-default-pool map", so I don't think it will break anything.

Reply:
for other command, such as unmap, list-mapped, no pool/image should be input.